code/modules/mob/inventory.dm

```diff
@@ -263,24 +270,27 @@ var/list/slot_equipment_priority = list( \
 		I.dropped(src)
 	return 1
 
+/mob/proc/drop_held()
+	unEquip(get_active_held_item())
```

Comment:
This implementation seems redundant with `unequip_item` above and also inconsistent with the name; did you intend for it to drop all held items instead?

Reply:
I have swapped `unequip_item` out for `unEquip(item)` or `drop_held` and removed it entirely, also renamed `drop_held` to `drop_held_items`.